packages/react-scripts/package.json

```diff
@@ -79,6 +79,7 @@
     "terser-webpack-plugin": "3.0.2",
     "ts-pnp": "1.2.0",
     "url-loader": "4.1.0",
+    "web-vitals": "^0.2.2",
```

Comment:
This could be a template dependency instead. There's nothing in react scripts itself that needs this.

Reply:
Thanks @ianschmitz, moved it to `template.json` for both templates (default and typescript). 

I also had to add it as a dev dependency in the root `package.json` (otherwise it wouldn't work locally). I hope I'm not doing something wrong here?

Reply:
@mrmckeb I think we ran into this with testing library? Probably need to fix that still.

Reply:
@housseindjirdeh We do now support `devDependencies` in templates, but that hasn't been released yet.
https://github.com/facebook/create-react-app/pull/8838

@ianschmitz I've added this to the agenda to discuss this week, and I can then try to make the changes. It'll only affect tests, so won't block the next release - but needs fixing, agreed.